Bynder/bynder-php-sdk
# API-16: Pre-release dev branch for support for FS style upload workflow (#34)

**This is a pre-release PR to be merged when the SDKs are tested against staging when [API-55](https://bynder.atlassian.net/browse/API-55) is resolved.**


* new FS upload

- new fs upload workflow
- remove old aws s3 dependencies
- tests

* tests, docs, readme fix

* documentation comments and code cleanup

* minor test fixes and more formatting

* suggested changes and formatting

-get ride of state variables
- cleanup RequestHandler
- format tests

* suggested changes- part 1

- remove fileID and correlationId return from uploadFile
- clean up doc comments
- clean up fileSha256 usage
-revert AbstractRequestHandler (no raw response needed anymore)
- add brandId non-null check
- test for invalid brandId

* suggested changes - part 2

- clean the header updation a little

* suggested changes - 1

- change filePath from a request param to a method param
- recycle code in tests
- fix saveMediaAsync for mediaId and brandId
- fix tests per the changes
- update sample and readme with new uploadFileAsync usage

* comments to save media for understanding

* suggested changes - 2

- prune upload intent for sdks
- remove server-ignored form_params
